gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceMessages.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.knox.gateway.i18n.messages.Messages;
 import org.apache.knox.gateway.i18n.messages.StackTrace;
 
-@Messages(logger="org.apache.knox.gateway.service.knoxsso")
+@Messages(logger="org.apache.knox.gateway.service.token")
```

Comment:
Might make sense to match the package name? `knoxtoken`

Reply:
I can buy that. I was surprised to see that it had been knoxsso :-O